pkg/reconciler/pingsource/resources/service_account.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/kmeta"
+
+	"knative.dev/eventing/pkg/apis/sources/v1alpha2"
+)
+
+// MakeServiceAccount creates a ServiceAccount object for the given PingSource
+func MakeServiceAccount(source *v1alpha2.PingSource, name string) *corev1.ServiceAccount {
```

Comment:
if we did not put the source - but the namesapce.

peraps this is useful on `pkg` ? I think this can be reused in contrib for some sources

Reply:
I can move it to pkg/reconciler/resources